### PR TITLE
Add Ubuntu/Debian setup guide for contributors

### LINKED ISSUE #2: Add a Miru app setup documentation for Ubuntu

### PR DESCRIPTION
**Pull request by aaj013**
Closes #2

### CODE REVIEW

File: docs/contributing-guide/setup/ubuntu_debian.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
+---
+id: ubuntu-debian
+title: Ubuntu 21.04/Debian 11
```
Comment (akhilgkrishnan) [inline docs/contributing-guide/setup/ubuntu_debian.md:3]:
```suggestion
title: Ubuntu/Debian
```

File: docs/contributing-guide/setup/ubuntu_debian.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+#### 3. Install [rvm](https://rvm.io/) and [nvm](https://github.com/nvm-sh/nvm#installing-and-updating)
+
+   (**Tip**: To allow `nvm` to automatically detect and change node versions for
+   your project as you `cd` into the directory follow
+   [this](https://github.com/nvm-sh/nvm#deeper-shell-integration))
+
+#### 4. Install ruby 3.1.1
+
+```bash
+rvm install 3.1.1
```
Comment (akhilgkrishnan) [inline docs/contributing-guide/setup/ubuntu_debian.md]:
```suggestion
rvm install $(cat .ruby-version)
```

File: docs/contributing-guide/setup/ubuntu_debian.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+```
+
+#### 5. Install Node 16.4.2(can be skipped if you followed the tip mentioned in (3) above)
+
+```bash
+nvm install 16.4.2
```
Comment (akhilgkrishnan) [inline docs/contributing-guide/setup/ubuntu_debian.md]:
```suggestion
nvm install $(cat .nvmrc)
```

**Review by akhilgkrishnan — CHANGES_REQUESTED** [review]

File: docs/contributing-guide/setup/ubuntu_debian.md (hunk 1 above)
Comment (aaj013) [reply docs/contributing-guide/setup/ubuntu_debian.md:3]: Just wanted to mention the versions as some of the steps may not be same for older/newer versions. How about mentioning it somewhere below the title? Can you put in suggestions?

**Review by akhilgkrishnan — APPROVED** [review]
Hi @aaj013  :wave:
Thanks for showing interest in contributing to Miru Web.
If you have any questions related to setting up Miru, feel free to join our Discord community here: [Discord](https://discord.gg/UABXyQQ82c)

**Pull request merged**